packages/foreman/rubygem-pg/rubygem-pg.spec

```diff
@@ -1,101 +1,103 @@
+# template: scl
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
-# Generated from pg-0.11.0.gem by gem2rpm -*- rpm-spec -*-
 %global gem_name pg
 
-Summary: A Ruby interface to the PostgreSQL RDBMS
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.21.0
-Release: 3%{?dist}
+Version: 1.1.4
+Release: 1%{?dist}
+Summary: Pg is the Ruby interface to the {PostgreSQL RDBMS}[http://www.postgresql.org/]
 Group: Development/Languages
 # Upstream license clarification (https://bitbucket.org/ged/ruby-pg/issue/72/)
 #
 # The portions of the code that are BSD-licensed are licensed under
 # the BSD 3-Clause license; the contents of the BSD file are incorrect.
 #
 License: (GPLv2 or Ruby) and PostgreSQL
-URL: http://bitbucket.org/ged/ruby-pg/
+URL: https://bitbucket.org/ged/ruby-pg
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
 Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby >= 2.0.0
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix_ruby}ruby
 BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}ruby-devel
+BuildRequires: %{?scl_prefix_ruby}ruby-devel >= 2.0.0
 BuildRequires: %{?scl_prefix_ruby}rubygems-devel
-BuildRequires: %{?scl_prefix_ruby}ruby
-BuildRequires: postgresql-server postgresql-devel
-# Introduced in F17.
-Obsoletes: %{?scl_prefix}ruby(postgres) <= 0.7.9-2010.01.28.2
 Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
-%{?scl:Obsoletes: ruby193-rubygem-%{gem_name}}
+# end specfile generated dependencies
+BuildRequires: postgresql-devel
 
 %description
-This is the extension library to access a PostgreSQL database from Ruby.
-This library works with PostgreSQL 7.4 and later.
+Pg is the Ruby interface to the {PostgreSQL
```

Comment:
Probably good to keep the old description

Reply:
no (as it would be lying about the PostgreSQL version), but I made the new description as close to the old as possible ;)